src/controllers/server.js

```diff
@@ -643,7 +641,7 @@ class Server extends EventEmitter {
         // on and check aganist the normal return path.
         if (!_.isNull(nonExistentPathRoot)) {
             if (_.startsWith(nonExistentPathRoot, dataPath)) {
-                return resolvedPath;
+                return nonExistentPathRoot;
```

Comment:
This seems to be correct.
`nonExistentPathRoot` should be replaced by `resolvedPath` everywhere instead though.

Reply:
I don't understand what do you mean.

Reply:
Instead of returning `nonExistentPathRoot` here, `resolvedPath` should be returned and also be assigned in [L624](https://github.com/pterodactyl/daemon/pull/91/files#diff-dbc455111cf309e73b959085e9b5b0adL624).

Reply:
Makes sense.